subsys/bluetooth/host/gatt.c

```diff
@@ -504,8 +504,26 @@ static bool cf_set_value(struct gatt_cf_cfg *cfg, const uint8_t *value, uint16_t
 	}
 
 	/* Set the bits for each octect */
-	for (i = 0U; i < len && i < last_byte; i++) {
-		cfg->data[i] |= value[i] & (BIT(last_bit + 1) - 1);
+	for (i = 0U; i < len && i <= last_byte; i++) {
+		/* The server shall support the feature and fulfill
+		 * all requirements if a client feature bit is set by
+		 * a client.
+		 */
+		if (IS_ENABLED(CONFIG_BT_GATT_CACHING)) {
+			WRITE_BIT(cfg->data[i], CF_BIT_ROBUST_CACHING,
```

Comment:
I think we should close this PR.
As we discussed in:
https://github.com/AyturkDuzen/auto-pts/pull/18
this is a PTS defect.